examples/cppgc-object.rs

```diff
@@ -58,10 +48,11 @@ fn main() {
           let id = args.get(0).to_rust_string_lossy(scope);
 
           let templ = v8::ObjectTemplate::new(scope);
-          templ.set_internal_field_count(2);
-
+          templ.set_internal_field_count(1);
```

Comment:
@littledivy may need to weigh in on whether this affects deno_core: https://github.com/denoland/deno_core/blob/main/core/cppgc.rs#L18

Reply:
yeah i have some changes for deno_core that i'll push up after this is released.

Reply:
Yeah, can safely switch to 1 internal field issue since manual embedder field tags are not required